client/src/pages/Contact.js

```diff
@@ -1,155 +1,143 @@
 import { useState } from "react";
 import emailjs from "emailjs-com";
+import {
+	Card,
+	CardContent,
+	Grid,
+	Stack,
+	TextField,
+	Button,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import "./Home.css";
 
-const initialState = {
-  name: "",
-  email: "",
-  message: "",
+const Result = () => {
+	return <Typography color="black">Message sent successfully!</Typography>;
 };
-export const Contact = (props) => {
-  const [{ name, email, message }, setState] = useState(initialState);
 
-  const handleChange = (e) => {
-    const { name, value } = e.target;
-    setState((prevState) => ({ ...prevState, [name]: value }));
-  };
-  const clearState = () => setState({ ...initialState });
+// const initialState = {
+// 	name: "",
+// 	email: "",
+// 	message: "",
+// };
```

Comment:
Can we please remove this commented code? Once removed this PR is good to be merged

Reply:
Sorry about that, I was supposed to ask the team if there was any benefit to have using state for the contact details variables. Unused code removed for now :)